ocserv-certificates-letsencrypt.md

```diff
@@ -0,0 +1,143 @@
+## Ocserv Certificates - letsencrypt
+
+Author: Mauro Gaspari  
+
+### Scope
+This recipe provides a deployment example of letsencrypt to provide ssl certificates for ocserv. Shorewall (ipv4) is used as firewall.  
```

Comment:
Are the firewall instructions necessary for the letsencrypt execution? If not would it make sense to simply refer to the shorewall instructions instead of duplicating it?  I'm thinking that the shorewall use here will limit the use of the guide to users of shorewall only. What do you think?

Reply:
This makes sense. 
There are some firewall steps that were not in the original shorewall recipe. Having said that, sure I can add a "letsencrypt" section in the shorewall recipe, and then point to it for firewall configuration. This means I will need to modify 2 recipes. If that is fine with you, I am ok with the change.

Reply:
Looks fine to me!